docusaurus/docs/dev-docs/configurations/admin-panel.md

```diff
@@ -42,6 +42,7 @@ The `./config/admin.js` file can include the following parameters:
 | `rateLimit.prefixKey`             | Prefix for the rate limiting key                                                                                                                                                                         | string        | `${userEmail}:${ctx.request.path}:${ctx.request.ip}`                                                                                |
 | `rateLimit.whitelist`             | Array of IP addresses to whitelist from rate limiting                                                                                                                                                    | array(string) | `[]`                                                                                                                                |
 | `rateLimit.store`                | Rate limiting storage location (Memory, Sequelize,  or Redis) and for more information please see the [`koa2-ratelimit documentation`](https://www.npmjs.com/package/koa2-ratelimit)                                                                                                                                                                                     | object        | `MemoryStore` |
+| `transfer.token.salt`                   | Salt used to generate [Transfer tokens](/dev-docs/data-management/transfer#generate-a-transfer-token)                                                                                 | string        | Random string                                                                                                                       |
```

Comment:
Should we add to the description that if no transfer token salt is defined, then the transfer features will be disabled? Or should this be part of the data management doc?

Reply:
It should be in both, actually. I'll update both, thanks!

Reply:
✅ [👉](https://github.com/strapi/documentation/pull/1551/commits/3cbe54e33399e11b9c1597f2e7f88caaa8158d89)